### PR TITLE
Add this branch name to CircleCI config to test build

### PR DESCRIPTION
**Pull request by juliatan**
# Description

[comment]: <> (Description of the feature or fix)
[comment]: <> (Include link to Figma designs, if relevant)
[comment]: <> (Include link to Notion ticket, if relevant)

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

## Screenshots

[comment]: <> (Please attach screenshots showing the change, if relevant)

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Checklist (for merger)

- [ ] Make sure the right merge strategy is chosen, commit merge into master and squash merge into develop.

### CODE REVIEW

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,5 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
 zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
\ No newline at end of file
+zipStorePath=wrapper/dists
+zipStorePath=wrapper/dists
```
Comment (aikewoody) [inline android/gradle/wrapper/gradle-wrapper.properties:7]: I think this duplicate line should be deleted

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "lint:fix": "yarn lint --fix",
     "metro-empty-cache": "rm -rf $TMPDIR/metro-cache",
     "mock-server": "ts-node --transpile-only ./mock-server/mockServer.ts --watch",
-    "postinstall": "ts-node --transpile-only ./scripts/postinstall.ts",
+    "postinstall": "cd ios && pod install",
```
Comment (aikewoody) [inline package.json:39]: If we are bypassing the postinstall script the file itself should be deleted as well as it's not being used anymore

File: .circleci/config.yml [hunk 3]
```diff
@@ -55,11 +55,13 @@ references:
   master_filter: &master_filter
     branches:
       only: master
+      
 
   develop_filter: &develop_filter
     branches:
       only:
         - develop
+        - julia/circleci-upgraded-version
```
Comment (aikewoody) [inline .circleci/config.yml:64]: Should this line of debug code stay here?

**Comment by aikewoody** [comment]
@juliatan Nice work! There was a minor merge conflict which I just solved

**Comment by bionara** [comment]
Using the actual branch now.

**Pull request closed without merging**